agent/agent.go

```diff
@@ -29,6 +29,8 @@ import (
 	"github.com/aws/amazon-ecs-agent/agent/engine"
 	"github.com/aws/amazon-ecs-agent/agent/engine/dockerclient"
 	"github.com/aws/amazon-ecs-agent/agent/engine/dockerstate"
+	eniWatcher "github.com/aws/amazon-ecs-agent/agent/eni"
+	"github.com/aws/amazon-ecs-agent/agent/eni/netlinkWrapper"
```

Comment:
Package names should not be camelCase or multiple words; see [here](https://blog.golang.org/package-names).